templates/macros/_vf-latest-news.jinja

```diff
@@ -0,0 +1,95 @@
+{#
+  Params:
+    title_text(string) (required): The title of the section, which also links to the main blog page.
+    tag_name (string) (required): The URL-friendly slug of a tag to filter the articles by.
+    tag_id (string) (required): The ID of a tag to filter the articles by.
```

Comment:
This patter will be very useful for future development, we can even add more options like this [one](https://github.com/canonical/canonical.com/blob/main/templates/shared/_blog-card.html)
in future to support more flexibility.

Reply:
I have purposely added the bare minimum for now to avoid it getting overly bloated. We can add to it as we go